docs/source/gravitysensors/tilt.rst

```diff
@@ -1,7 +1,76 @@
-====================
+Tilt Setup
+==========
 
-.. todo:: Write the Tilt Sensor Setup page
+To associate your a Tilt with Fermentrack, simply place the Tilt near the Raspberry Pi you are using at an angle (not vertical or horizontal) and it will automatically detect it (The Tilt enters a sleep mode when horizontal or vertical).
 
+#. From the main Fermentrack dashboard, click on "Add new gravity sensor"
+#. Click on the Tilt icon
+#. Name the Tilt hydrometer you are adding, select the temperature units you wish to use, and as select the color for easy identification. You also want to specify if you are using a direct Bluetooth connection to your Pi, or using a TiltBridge bluetooth to WiFi adapter.
+
+You will return to the Dashboard and Fermentrack will notify you that a new tilt sensor was added.
+
+Tilt Settings
+-------------
+
+After adding the Tilt to Fermentrack, you are returned to the Gravity Sensors dashboard. Right now, the Tilt is running in unassociated mode in Fermentrack, meaning it is handled as its own device- not integrated with the BrewPi controller. Clicking on the device you just added will bring you into a screen where you can further configure and customize that Tilt sensor. From this page, you can `Manage Sensor`_, `Load Prior Log`_, and `Assign Device`_.
+
+Manage Sensor
+~~~~~~~~~~~~~
+
+From the sensor dashabord, click on Manage Sensor. This page will show you the current settings running on the Tilt sensor, allow you to add manual calibration points, as well as allow you to perform a `Guided Calibration`_. The B, C, and D coefficients shown are automatically applied/updated after running a guided calibration. You can also remove this sensor from Fermentrack at the bottom of the page.
+
+Load Prior Log
+~~~~~~~~~~~~~~
+
+This button allows you to view prior logs generated by the Tilt sensor. This will bring up a window with available logs. You can click on the log name to view the graph in the browser, or click "View Full CSV" to download the csv file that holds the raw data for the graph. You can also choose "Delete" to delete the log. Viewing a previous graph does not stop Fermentrack from collecting data from the Tilt on the active log. You can return to the active log by clicking the "Load Prior Log" button and then "Return to Active Log".
+
+Assign Device
+~~~~~~~~~~~~~
+See `Integrating with BrewPi`_
+
+Logging/Visualization
+---------------------
+
+To start logging data points from your Tilt, click below the device name in the orange box. Enter a name for your log in the text field and click "Create Log & Start Logging". This will create a blank graph where your data points will begin to populate after a few moments of being collected. The page does not refresh automatically so you will have to manually refresh to see updates.
+
+Add Data Point
+~~~~~~~~~~~~~~
+
+This button allows you to manually add a data point to the graph. What you enter here will appear on the graph at the time you enter it.
```

Comment:
I don't have an install of Fermentrack handy, but if this is the feature I think it is, its primary purpose is to enable adding annotations to the graph ("added yeast", "racked", etc.)